enterprise/frontend/src/metabase-enterprise/plugins.js

```diff
@@ -75,3 +75,13 @@ const activateEEPlugins = () => {
 };
 
 activateEEPlugins();
+
+const activatePluginsSystem = () => {
```

Comment:
i should probably find better names

this activates the plugin system

maybe plugins should "run" instead of "activate"